### DISCUSSION TITLE
Markering av chackboks

### DISCUSSION THREAD

**Discussion by mariusthedesigner** [fremtind/jokul#1863] [Annet]
Jeg foreslår noen små endringer på checkboks-komponenten. Noen små justeringer på avstander (Figma) og tydeligere markering ved avhuking.
<img width="692" alt="Skjermbilde 2021-03-03 kl  08 54 07" src="https://user-images.githubusercontent.com/59652564/109775524-e55fbb80-7c01-11eb-9c7a-aed30f37c4c8.png">
<img width="663" alt="Skjermbilde 2021-03-03 kl  08 54 01" src="https://user-images.githubusercontent.com/59652564/109775529-e690e880-7c01-11eb-94de-e0d2c5cc70b4.png">

**Comment by lfbergee** [comment]
Dagens for referanse
![image](https://user-images.githubusercontent.com/46530368/109780257-20b0b900-7c07-11eb-92d3-fa77dca9500e.png)

**Comment by vegardsoerlie** [comment]
Bra forslag, støttes 👍 

Tekstinput og select er også rundet av så gir mening å runde av denne også imo.

**Reply by mariusthedesigner** [reply]
Ja, to piksler avrunding. Gir et mer helhetlig inntrykk, dog veldig subtilt, men bidrar til å følge egne prinsipper :)

**Comment by lfbergee** [comment]
Har ikke noe problem med avrundinga eller bold ikonet, syns det er fine forbedringer.

Litt mer usikker på bold boks, bør ikke da også radiobuttons være bold når den er selected? Dette kan jo dere #designers mye bedre enn meg.

**Reply by mariusthedesigner** [reply]
Jo, radiobutton burde også følge samme prinsipp om vi vil gå for en slik løsning :)

**Reply by lfbergee** [reply]
Okei. Siste ting, hva gjøres med checkikonet https://jokul.fremtind.no/komponenter/icons ?

**Reply by mariusthedesigner** [reply]
Jeg har ikke vært med å spekke opp ikoner, så kanskje høre med Lou eller Mona? Jeg ser at om man bare endrer tykkelse blir ikke check-ikonet så tydelig, og trenger en «hale» som er en til to piksler lengre :)

**Reply by lfbergee** [reply]
Det er ikke det faktiske ikonet som er i bruk i komponenten, så er ikke en teknisk begrensning, tenker bare på hvordan det henger sammen.

**Comment by Murstam** [comment]
Et annet alternativ for å gjøre den mer tydelig kan være å fylle checkboksen også. Da kan man også droppe å bolde teksten tenker jeg.
<img width="695" alt="Skjermbilde 2021-04-19 kl  12 22 09" src="https://user-images.githubusercontent.com/47025023/115221543-13de1b00-a10a-11eb-9bd8-df2d5cb54cba.png">
